# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/CreateElementTransaction.cpp
# Commit: d3816f82c0b9
# Full Hash: d3816f82c0b9d4ceca0bb2ad177694164ed0921f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-04-03 21:40:10
# Regressor Bug: 1619914
# File Overlap Count: 1
# Description:
#   Bug 1619914 - part 2: Mark transaction class methods and their caller methods as `MOZ_CAN_RUN_SCRIPT r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69303
# ==============================================================================

diff -r 876b1031a6ae -r d3816f82c0b9 editor/libeditor/CreateElementTransaction.cpp
--- a/editor/libeditor/CreateElementTransaction.cpp	Fri Apr 03 01:38:22 2020 +0000
+++ b/editor/libeditor/CreateElementTransaction.cpp	Fri Apr 03 01:39:36 2020 +0000
@@ -69,8 +69,7 @@
 NS_INTERFACE_MAP_BEGIN_CYCLE_COLLECTION(CreateElementTransaction)
 NS_INTERFACE_MAP_END_INHERITING(EditTransactionBase)
 
-MOZ_CAN_RUN_SCRIPT_BOUNDARY NS_IMETHODIMP
-CreateElementTransaction::DoTransaction() {
+NS_IMETHODIMP CreateElementTransaction::DoTransaction() {
   if (NS_WARN_IF(!mEditorBase) || NS_WARN_IF(!mTag) ||
       NS_WARN_IF(!mPointToInsert.IsSet())) {
     return NS_ERROR_NOT_INITIALIZED;